examples/windows/kubernetes-hyperv.json

```diff
@@ -5,37 +5,42 @@
       "orchestratorType": "Kubernetes",
       "orchestratorRelease": "1.15",
       "kubernetesConfig": {
-               "apiServerConfig" : {
-                 "--feature-gates": "HyperVContainer=true"
-               },
-               "kubeletConfig" : {
-                    "--feature-gates": "HyperVContainer=true"
-                }
-          }
+        "networkPlugin": "azure",
+        "containerRuntime": "containerd",
+        "windowsContainerdURL": "https://k8swin.blob.core.windows.net/k8s-windows/containerd/containerplat-aks-test-0.0.8.zip"
+      }
     },
     "masterProfile": {
       "count": 1,
       "dnsPrefix": "hypervtest",
       "vmSize": "Standard_D2_v3"
     },
     "agentPoolProfiles": [
-	    {
-		"name": "windowspool",
-		"count": 2,
-		"vmSize": "Standard_D2_v3",
-		"availabilityProfile": "AvailabilitySet",
-		"osType": "Windows",
-		"osDiskSizeGB": 128,
-		"extensions": [
-		    {
-                        "name": "winrm"
-		    }
-		]
-	    }
+      {
+        "name": "windowspool",
+        "count": 2,
+        "vmSize": "Standard_D4s_v3",
+        "availabilityProfile": "AvailabilitySet",
+        "osType": "Windows",
+        "osDiskSizeGB": 128
+      }
     ],
     "windowsProfile": {
       "adminUsername": "azureuser",
-      "adminPassword": "replacepassword1234$"
+      "adminPassword": "replacepassword1234$",
+      "enableAutomaticUpdates": false,
+      "sshEnabled": true,
+      "windowsPublisher": "MicrosoftWindowsServer",
+      "windowsOffer": "WindowsServer",
+      "windowsSku": "Datacenter-Core-2004-with-Containers-smalldisk",
+      "imageVersion": "latest",
+      "windowsRuntimes": {
```

Comment:
What do you think about moving this under the kubernetesConfig instead of having this be in WindowsProfile?
My thinking is
1. The rest of the runtime settings (docker vs containerd, containerd URl etc) are all there so the settings would stay together
1. We might be moving these settings to a place where they can be different for each agent pool. I don't think we know how that will look yet.

Reply:
I had a similar thought. I am not sure if there is need from the containerd side on linux with a runtime like kata?  @jackfrancis @mboersma any thoughts?

Reply:
OK - lets revisit when we look at #3687

Reply:
What does this additional configuration inform? Does it act as input for producing a working containerd configuration?